src/Features/Core/Portable/Snippets/RoslynLSPSnippetConverter.cs

```diff
@@ -119,7 +119,7 @@ private static async Task<TextChange> ExtendSnippetTextChangeAsync(Document docu
 
             var documentText = await document.GetTextAsync(cancellationToken).ConfigureAwait(false);
             var newString = documentText.ToString(extendedSpan);
-            var newTextChange = new TextChange(new TextSpan(extendedSpan.Start, 0), newString);
+            var newTextChange = new TextChange(TextSpan.FromBounds(extendedSpan.Start, extendedSpan.End), newString);
```

Comment:
isn't `TextSpan.FromBounds(extendedSpan.Start, extendedSpan.End)` exactly the same as `extendedSpan`?